rocAL/rocAL/include/pipeline/ring_buffer.h

```diff
@@ -72,13 +72,12 @@ class RingBuffer
     void increment_write_ptr();
     bool full();
     const unsigned BUFF_DEPTH;
-    unsigned _sub_buffer_size;
+    std::vector<size_t> _sub_buffer_size;
     unsigned _sub_buffer_count;
     std::mutex _lock;
     std::condition_variable _wait_for_load;
     std::condition_variable _wait_for_unload;
     std::vector<std::vector<void*>> _dev_sub_buffer;
-    std::vector<void*> _host_master_buffers;
```

Comment:
Why have we removed this line ? The concept of master buffer has been removed ?

Reply:
Yes the master buffers are removed from ring buffer